src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You should provide a generic font family as a backup. For example: `font-family: Roboto, Arial, sans-serif;` should be `font-family: Roboto, Arial, 'Helvetica Neue', Helvetica, sans-serif;`.